taiga-front-dist/README.md

```diff
@@ -52,15 +87,31 @@ taigaback:
 taigafront:
   image: htdvisser/taiga-front-dist:stable
   hostname: dev.example.com
+  environment:
```

Comment:
YAML dictionary format used here to make the multiline certificate values easier to maintain.